Assets/Aircraft Physics/Example/Scripts/AirplaneController.cs

```diff
@@ -46,9 +46,13 @@ private void Update()
         if (Input.GetKey(KeyCode.Space))
         {
             SetThrust(thrustPercent + thrustControlSensitivity);
+        }
+          if (Input.GetKey(KeyCode.F))
+        {
+            SetThrust(0);
```

Comment:
This is also unnecessary, press shift and space at the same time to lower the thrust.